addon/-private/system/model/internal-model.js

```diff
@@ -885,14 +885,14 @@ export default class InternalModel {
         let rel = this._relationships.get(name);
 
         rel.clear();
-        // rel.removeInverseRelationships();
+        rel.removeInverseRelationships();
```

Comment:
hmm yeah it looks like the unload tests assert against the id map and the relationship payloads, but not against relationships (either our side or inverse).